utilities/roslz4/src/lz4s.c

```diff
@@ -47,6 +47,16 @@
 #define DEBUG(...)
 #endif
 
+// Make sure the LZ4 library has the function we need
+// Note that the later versions of LZ4 contain a different macro that does this version math, but the earlier
+// versions lack it.
+#define LZ4_VERSION (LZ4_VERSION_MAJOR * 100 * 100 + LZ4_VERSION_MINOR * 100 + LZ4_VERSION_RELEASE)
+#if LZ4_VERSION >= 10700
```

Comment:
>  That tag identifies itself as version 1.7.0 which is also the same for the earlier tag `r129`.

As mentioned before since an older release (`r129`) identifies itself with this version too I don't think it can be used for the check.

Reply:
Oh, apologies, I misunderstood your issue. I will get the first post-1.7.0 tag, which I believe is 1.7.3.

Reply:
`r131` identifies as 1.7.1. Would that be more precise?

Reply:
Yes, it would.